symfony/framework-bundle/6.2/config/services.yaml

```diff
@@ -10,6 +10,11 @@ services:
     _defaults:
         autowire: true      # Automatically injects dependencies in your services.
         autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.
+        # Bind arguments globally
+        # https://symfonycasts.com/screencast/symfony-fundamentals/global-bind
```

Comment:
I'd prefer a reference to the Symfony docs.

Reply:
link updated